gui/src/Users/reducers/users.reducer.ts

```diff
@@ -17,19 +17,54 @@
 
 import * as _ from 'lodash';
 import {
-  RECEIVE_PERMISSIONS_WITH_ROLES, RECEIVE_ROLE_NAMES, RECEIVE_PROJECTS, RECEIVE_USERS, RECEIVE_SAVE_PERMISSIONS, RESET_UPDATES,
-  RECEIVE_USERS_WITH_ROLES_AND_PROJECTS, FILTER_SELECTIONS_CHANGED
+  RECEIVE_PERMISSIONS_WITH_ROLES, RECEIVE_ROLE_NAMES, RECEIVE_PROJECTS, RECEIVE_USERS, RECEIVE_SAVE_PERMISSIONS,
+  RESET_UPDATES, RECEIVE_USERS_WITH_ROLES_AND_PROJECTS, FILTER_SELECTIONS_CHANGED, ENTER_NEW_ROLE, EXIT_NEW_ROLE, ENTER_NEW_USER,
+  EXIT_NEW_USER, RECEIVE_CREATE_ROLE, RECEIVE_WORKGROUPS_FOR_IDENTITY
 } from '../actions/users.actions';
 
 let initialState = {
-  permissionWithRoles: [],
+  permissionsWithRoles: [],
   roles: [],
   projects: [],
   updates: [],
+  createNewUserIsEntered: false,
+  createNewRoleIsEntered: false
 };
 
 export const usersReducer = (state: any = initialState, action: any) => {
   switch (action.type) {
+    case RECEIVE_WORKGROUPS_FOR_IDENTITY :
```

Comment:
Some of your case blocks have are wrapped in curly braces and some aren't. I have no opinion either way as to which style you use so long as it's consistent.